public/app/plugins/datasource/loki/components/LokiQueryEditor.tsx

```diff
@@ -104,9 +104,10 @@ export const LokiQueryEditor = React.memo<LokiQueryEditorProps>((props) => {
       query.queryType,
       previousQueryType
     );
-    if (update) {
+    // Time range check duplicated to satisfy typescript. Already checked in shouldUpdateStats.
+    if (update && timeRange) {
```

Comment:
It think it would be even better if `update` was renamed to `shouldUpdate` for readability.

Reply:
I agree, updated in https://github.com/grafana/grafana/pull/78603/commits/b45d98a826c25fc70e5c80559699f460b832b4c2